person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_reader :id, :name, :age
```

Comment:
- [ ] It would better to make  :name and :age instance variables the only ones that should be able to be modified, kindly define them as attr_accessors, based on the requirements  only the id should be defined as an `attr_reader`